### DISCUSSION TITLE
No error and no display of a shape

### DISCUSSION THREAD

**Discussion by tito125** [XusinboyBekchanov/VisualFBEditor#620] [Q&A]
Hello and thank you for this application

I tried to make a shape and display it with WASM, but I have no error. 
![01](https://github.com/XusinboyBekchanov/VisualFBEditor/assets/53943846/a875265e-0b12-4f61-be68-ddb963c7c202)
I don't understand. Attached is a screenshot.
![02](https://github.com/XusinboyBekchanov/VisualFBEditor/assets/53943846/394c83ef-2ac0-4a8d-bcfc-70d230427074)

The form1.HTML must be created somewhere?

**Comment by XusinboyBekchanov** [comment]
>Hello and thank you for this application

Hello

>I tried to make a shape and display it with WASM, but I have no error.

I see from the screenshot that you didn’t compile it, but just launched it.

>The form1.HTML must be created somewhere?

To create this file you need to compile the program, to do this you can press F5 or the double arrow image:
![image](https://github.com/XusinboyBekchanov/VisualFBEditor/assets/32607344/1b00391a-5969-4916-ba50-92fc5dd526fd)

First you need to set up your environment using this link:
https://freebasic.net/forum/viewtopic.php?p=277649#p277649

**Comment by tito125** [comment]
Hello; 
I've tried several times, but in the pyton script "emsdk", there must be a problem in the download links and for the node install.

I understand that you need "Emc" in ". exe" for FBC to use it, and that's why there's an error in


**`01:44:57: compiling C :  C:\Users\PC1\Downloads\Basic code\Application\VisualFBEditor.1.3.5\emcc -c -nostdlib -nostdinc -Wall -Wno-unused-label -Wno-unused-function -Wno-unused-variable -Wno-warn-absolute-paths -Wno-main -Werror-implicit-function-declaration -fno-strict-aliasing -fno-math-errno -fwrapv -fno-exceptions -fno-asynchronous-unwind-tables -funwind-tables -Wno-format "Form1.c" -o "Form1.o"

Error: Couldn't Create Process

01:44:57: linking :      C:\Users\PC1\Downloads\Basic code\Application\VisualFBEditor.1.3.5\**emcc** -o "Form1.html" -O0 -Wno-warn-absolute-paths -s CASE_INSENSITIVE_FS=1 -s TOTAL_MEMORY=67108864 -s ALLOW_MEMORY_GROWTH=1 -s DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=$stringToNewUTF8 -s RETAIN_COMPILER_SETTINGS=1 --shell-file "C:\Users\PC1\Downloads\Basic code\Application\VisualFBEditor.1.3.5/.\Controls\MyFbFramework\mff\Web\mff.html" --post-js "C:\Users\PC1\Downloads\Basic code\Application\VisualFBEditor.1.3.5\Compilers\FreeBASIC-1.10.1-win64\lib\js-asmjs\fb_rtlib.js" --post-js "C:\Users\PC1\Downloads\Basic code\Application\VisualFBEditor.1.3.5\Compilers\FreeBASIC-1.10.1-win64\lib\js-asmjs\termlib_min.js" -L"C:\Users\PC1\Downloads\Basic code\Application\VisualFBEditor.1.3.5\Compilers\FreeBASIC-1.10.1-win64\lib\js-asmjs" -L"." "Form1.o" -lfb -lfb  -s ASYNCIFY=1 -s ERROR_ON_UNDEFINED_SYMBOLS=0 -s WASM=1 -s EXPORTED_FUNCTIONS="['_ONSTART', '_ONLOAD', '_ONCHANGE', '_ONCLICK', '_ONDBLCLICK', '_ONGOTFOCUS', '_ONLOSTFOCUS', '_ONKEYDOWN', '_ONKEYPRESS', '_ONKEYUP', '_ONMOUSEENTER', '_ONMOUSEDOWN', '_ONMOUSEMOVE', '_ONMOUSEUP', '_ONMOUSELEAVE', '_ONMOUSEWHEEL', '_ONUNLOAD']" --post-js C:\Users\PC1\Downloads\Basic code\Application\VisualFBEditor.1.3.5/.\Controls\MyFbFramework\mff\Web\mff.js
Error: Couldn't Create Process`**

**Comment by XusinboyBekchanov** [comment]
The link I provide mentions this in the paragraph "Hook for 'emcc.bat'"
